ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts

```diff
@@ -93,17 +95,23 @@ export class FooterComponent implements OnInit {
         private dialog: MatDialog,
         private analytics: AnalyticsEventsService,
         private windowRef: WindowRef,
-        @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
+        @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService,
+        private ngxTranslate: NGXTranslateService) { }
 
     public ngOnInit(): void {
         this.phone = this.toolkitConfiguration.phone;
         this.email = this.toolkitConfiguration.infoEmail;
         this.phoneHref = `tel:${this.toolkitConfiguration.phone}`;
         this.emailHref = `mailto:${this.toolkitConfiguration.infoEmail}`;
-        this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
+        // this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
         this.twitterUrl = `https://twitter.com/${this.toolkitConfiguration.twitterAccountId}`;
         this.countMeInUrl = this.toolkitConfiguration.countMeInUrl;
         this.blogUrl = this.toolkitConfiguration.blogUrl;
+        const fbLinkKey = 'Toolkit.Footer.FacebookLink';
+        this.facebookUrl$ = this.ngxTranslate.getTranslation(fbLinkKey).pipe(
+            map(transVal => transVal === fbLinkKey ?
+                `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}` : transVal)
+        );
```

Comment:
The footer component is shared, so tried to make this change backwards compatible. We look for link in translation file and if not found there, we use the toolkit configuration.